dependencies.list

```diff
@@ -1,5 +1,5 @@
 PACKAGE_NAME=realm-js
 VERSION=2.0.0-rc9
 REALM_CORE_VERSION=3.2.1
-REALM_SYNC_VERSION=2.0.0-rc21
+REALM_SYNC_VERSION=2.0.0-rc22
 REALM_OBJECT_SERVER_VERSION=2.0.0-alpha.32
```

Comment:
Title says rc23, you use rc22?
update to ROS to alpha.35

Reply:
@kneth

Reply:
Sync's "rc23" release ended up being versioned as rc22 everywhere except the change log entry.

Reply:
@bmunkholm Sorry for the title being unclear. Btw, this is for a release and all sync tests are currently disabled on the `2.0.x` branch so we don't need to update ROS.